README.md

```diff
@@ -6,7 +6,9 @@ A module with convenience functions for writing Python code that interacts with
 
 # Installation
 
-Just `pip install ezomero` and you should be good to go! The repo contains the specific package versions we test `ezomero` with in `setup.py`, but any Python>=3.8 and latest `omero-py` and `numpy` _should_ work -  note that this package is in active development!
+ezomero's dependencies are easily pip-installable from PyPI, except for `zeroc-ice==3.6.5`. For those, we recommend pip-installing using one of the [wheels](https://github.com/glencoesoftware/zeroc-ice-py-linux-x86_64/releases) provided by Glencoe Software (use the one compatible  with your OS/Python version - link provided is for Linux wheels).
```

Comment:
Maybe this is a better link?
```suggestion
ezomero's dependencies are easily pip-installable from PyPI, except for `zeroc-ice==3.6.5`. For those, we recommend pip-installing using one of the [wheels](https://www.glencoesoftware.com/blog/2023/12/08/ice-binaries-for-omero.html) provided by Glencoe Software (use the one compatible  with your OS/Python version - link provided is for Linux wheels).
```
If someone wants to run on macOS or whatnot.

Reply:
oh dang I only saw this right after I merged it :/